tests/linalg_test.py

```diff
@@ -1480,6 +1480,32 @@ def testRsf2csf(self, shape, dtype):
                               args_maker, tol=tol)
       self._CompileAndCheck(jsp.linalg.rsf2csf, args_maker)
 
+  @parameterized.named_parameters(
+        jtu.cases_from_list({
+            "testcase_name":
+            "_shape={}_disp={}".format(jtu.format_shape_dtype_string(shape, dtype), disp),
+            "shape": shape, "dtype": dtype, "disp": disp
+        } for shape in [(1, 1), (5, 5), (20, 20), (50, 50)]
+            for dtype in float_types + complex_types
+              for disp in [True, False]))
+  @jtu.skip_on_devices("gpu", "tpu")
```

Comment:
Why skip on gpu and tpu? Do we not expect this function to work correctly on those backends? If so we should mention this in the documentation. If not, we should test it.

Reply:
`funm` uses `jax.scipy.linalg.schur` and my understanding is that Schur decomposition is currently implemented only for a CPU backend.

This also affects other methods such as `jax.scipy.linalg.sqrtm` which currently rely on Schur decomposition (and could affect yet-to-be-implemented methods such as `logm` if the same algorithm as SciPy is followed). 

I can add a note to the docstring explaining this (perhaps we'd want something generic that can be attached to other methods that may rely on Schur decomposition?).

Reply:
Makes sense. It would be worth adding a comment to that effect in the line where you skip the tests.